.github/workflows/rust.yml

```diff
@@ -165,11 +189,44 @@ jobs:
         run: |
           git diff --staged --no-ext-diff --quiet --exit-code
 
+  publish-release:
+    name: Check version and publish release
+    runs-on: ubuntu-latest
+    needs:
+      ["linux", "mac", "windows"]
+    if: ${{ !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/download-artifact@v4
+        with:
+          merge-multiple: true
+      - id: check-tag
+        if: ${{ github.ref == 'refs/heads/main' && github.event_name == 'push' }}
+        run: |
+          RELEASE_TAG=mozjs-sys-v$(cargo metadata --format-version=1 --no-deps | jq -r '.packages[] | select(.name == "mozjs_sys") | .version')
```

Comment:
```suggestion
          RELEASE_TAG=mozjs-sys-v$(cargo pkgid -p mozjs_sys | cut -d "@" -f2)
```

Reply:
cargo pkgid requires Cargo.lock. I'll see if I can move this step to previous building job.

Reply:
> cargo pkgid requires Cargo.lock

Didn't know that. Then it might just be better what you wrote originally. Sorry for trouble.

Reply:
Okay I'll revert it.